content/en/examples/pods/storage/projected-clustertrustbundle.yaml

```diff
@@ -4,12 +4,14 @@ metadata:
   name: sa-ctb-name-test
 spec:
   containers:
-  - name: container-test
+  - command:
+    - sleep
+    - "3600"
     image: busybox
-    command: ["sleep", "3600"]
+    name: container-test
     volumeMounts:
-    - name: token-vol
-      mountPath: "/root-certificates"
+    - mountPath: /root-certificates
```

Comment:
This is subjective depending on the person, but I have no issues with the arrangement.